docs/quickstart.md

```diff
@@ -0,0 +1,411 @@
+# Getting Started
+
+**Primer3-py** is designed to be simple to install and use.
+
+```{contents}
+```
+
+## Requirements
+
+**Primer3-py** is built and tested on Mac OS X and Linux systems; we do not
+provide official Windows support. Python versions 3.8 - 3.11 are supported.
+
+## Installation
+
+If you want to install the latest stable build of **Primer3-py**, you can
+install it using `pip`:
+
+```bash
+$ pip install primer3-py
+```
+
+Or via `conda`:
+
+```bash
+$ conda install primer3-py
+```
+
+## Thermodynamic analysis
+
+The thermodynamic bindings include support for **Tm, homodimer, heterodimer,
+hairpin,** and **3' end stability calculations**:
+
+```{eval-rst}
+.. py:function:: calc_tm(seq, mv_conc=50, dv_conc=0, dntp_conc=0.8, \
+                        dna_conc=50, dmso_conc=0.0, dmso_fact=0.6, \
+                        formamide_conc=0.8, annealing_temp_c=-10.0, \
+                        max_nn_length=60, tm_method='santalucia', \
+                        salt_corrections_method='santalucia')
+
+    Calculates the melting temperature of a DNA sequence, ``seq``. Returns the
+    melting temperature (C) as a float::
+
+        >>> primer3.calc_tm('GTAAAACGACGGCCAGT')
+        49.16808228911765
+
+    Note that NN thermodynamics will be used to calculate the Tm of sequences
+    up to 60 bp in length, after which point the following formula will be
+    used::
+
+        Tm = 81.5 + 16.6(log10([mv_conc])) + 0.41(%GC) - 600/length
+
+```
+
+```{eval-rst}
+.. py:function:: calc_hairpin(seq, mv_conc=50.0, dv_conc=0.0, dntp_conc=0.8, \
```

Comment:
Might be a follow up effort to take a pass at updating these docs -- e.g., `calc_hairpin` now has an `output_structure` kwarg.

Reply:
Tbh section of the `quickstart.md` I think needs a refactor.  I'll update this with the new args, but lets reconsider the whole of the docs in a follow up.

Reply:
Sounds good to me